tests/test_lens.py

```diff
@@ -46,324 +49,49 @@
 ]
 
 
-@pytest.fixture(scope="class")
-def init_lens(
-    classification_model,
-    classification_assessment_data,
-    classification_train_data,
-    request,
-):
-    gov = Governance()
-    my_pipeline = Lens(
-        model=classification_model,
-        assessment_data=classification_assessment_data,
-        training_data=classification_train_data,
-        governance=gov,
-    )
-    request.cls.pipeline = my_pipeline
+@pytest.fixture(scope="function")
+def temp_file(tmp_path):
```

Comment:
It's a pre existing fixture from pytest, it can be referred to without initializing it. It takes care of dumping everything after the test is ended. Similar to tempfile, but pytest native. See https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html